schema/base/land_cover.yaml

```diff
@@ -0,0 +1,50 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Land Cover Schema
+description: representation of the Earth's natural surfaces 
+type: object
+properties:
+  id:
+    "$ref": ../defs.yaml#/$defs/propertyDefinitions/id
+  geometry:
+    description: classification of surface, can correspond to the 'natural' tag from OpenStreetMap
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
+    required:
+      - subType
+    properties:
+      subType:
+        description: type of surface represented
+        type: string
+        enum:
+          - barren
+          - crop
+          - forest
+          - grass
+          - mangrove
+          - moss
+          - shrub
+          - snow
+          - urban
+          - wetland
+      cartography:
+        suggested_min_zoom:
+          description: recommended minimum tile zoom
+          type: number
+          minimum: 1
+          maximum: 24
+        suggested_max_zoom:
+          description: recommended maximum tile zoom
+          type: number
+          minimum: 1
+          maximum: 24
+        suggest_hidden:
+          description: true/false if the feature is suggested to be hidden for cartographic display
+          type: boolean
```

Comment:
If the cartography property will be shared by divisions (for suggesting hiding disputed) this will have to move the top-level defs and then referred here.

I think suggest_hidden won't be used at all for land_cover?

Reply:
agreed, will update shortly

Reply:
From the [2024-04-17](https://github.com/OvertureMaps/schema-wg/issues/293) schema TF meeting, seems like we need to drop `suggest_hidden`, define the `cartography` property as a "container" in the main `defs.yaml` and then include it into this schema by reference in the `allOf` clause up around line 17...

----

I realized I was wrong in proposing mix-and-match sub-properties in `cartography` per feature type. That won't work because it will probably break the Parquet schema generator.